Tiltfile

```diff
@@ -0,0 +1,28 @@
+###################
```

Comment:
These seem to map 1:1 (more or less) to the CLI equivalent of docker or helm, so I didn't document them here.